.github/workflows/timqi.yml

```
@@ -0,0 +1,73 @@
+  workflow_dispatch:
+
+jobs:
+  sync_and_build:
+    name: Sync latest commits from upstream repo & Build docker image
+    runs-on: ubuntu-latest
+    if: ${{ github.event.repository.fork }}
+
+    steps:
+      # Step 1: run a standard checkout action
+      - name: Checkout target repo
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)